code/modules/organs/limbs.dm

```diff
@@ -410,7 +410,7 @@
 		update_wounds()
 
 	//Bone fractures
-	if(CONFIG_GET(flag/bones_can_break) && brute_dam > min_broken_damage * CONFIG_GET(number/organ_health_multiplier) && !(limb_status & LIMB_ROBOT))
+	if(CONFIG_GET(flag/bones_can_break) && brute_dam > min_broken_damage * CONFIG_GET(number/organ_health_multiplier) && !(limb_status & LIMB_ROBOT) && !(SSticker.mode?.flags_round_type & MODE_NO_PERMANENT_WOUNDS))
```

Comment:
Should put this check inside fracture instead of here, since there's other things that break limbs directly.

Reply:
the other cases are from surgery mostly, so i think it's fine

Reply:
BHJ does in its overdose, is the other one I'm familiar with, but it'd be tricky to trigger that without also repairing them.